packages/libslirp/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://gitlab.freedesktop.org/slirp/libslirp
+TERMUX_PKG_DESCRIPTION="A general purpose TCP-IP emulator."
+TERMUX_PKG_LICENSE="BSD 3-Clause"
+TERMUX_PKG_LICENSE_FILE="COPYRIGHT"
```

Comment:
COPYRIGHT is in the [list of names](https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_install_license.sh#L43) that are search for [BSD 3-Clause](https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_install_license.sh#L36), so should be no need to specify TERMUX_PKG_LICENSE_FILE

Reply:
I updated it